v1/pipeline/steps/cd/k8s-rolling-rollback-step-info.yaml

```diff
@@ -3,48 +3,18 @@ allOf:
 - $ref: ../../common/step-spec-type.yaml
 - type: object
   properties:
-    commandFlags:
+    flags:
+      description: command_flags
       oneOf:
-      - type: array
-        items:
-          $ref: k8s-step-command-flag.yaml
-      - type: string
-        pattern: ^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$
-        minLength: 1
-    delegateSelectors:
+        - type: array
+          items:
+            $ref: k8s-step-command-flag.yaml
+        - type: string
+    delegates:
+      $ref: ../../common/delegate.yaml
+    pruning:
```

Comment:
In k8s-rolling-deploy step we are calling this `prune`.
File name: `k8s-rolling-step-info.yaml`

Reply:
done